tools/python/insert_localizations.py

```diff
@@ -0,0 +1,125 @@
+import os
+import re
+import sys
+import xml.etree.ElementTree as ET
+
+# Android config
+android_base = "../../android/app/src/main/res"
+android_file = "strings.xml"
+
+# iOS config
+ios_base = "../../iphone/Maps/LocalizedStrings"
+ios_file = "Localizable.strings"
+
```

Comment:
- Now we have android/app/ and android/sdk/ paths for strings.
- Also types_strings.xml
- Also LocalizableTypes.strings